File: ePozoriste.Model/Karta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ePozoriste.Model
{
    public class Karta
    {
        public int KartaId { get; set; }
        public bool Aktivna { get; set; }
        public int? TerminId { get; set; }
        public virtual Termin? Termin { get; set; }
        public int BrojSjedista { get; set; }
        public string BrojReda { get; set; }
        public string Sjediste { get; set; }


    }
}

File: epozoriste_mobile/lib/models/karta.dart
import 'package:epozoriste_mobile/models/models.dart';
import 'package:json_annotation/json_annotation.dart';

part 'karta.g.dart';

@JsonSerializable()
class Karta {
  int kartaId;
  bool aktivna;
  int brojSjedista;
  String brojReda;
  String sjediste;
  int? terminId;
  Termin termin;

  Karta(
      {required this.kartaId,
      required this.aktivna,
      required this.brojSjedista,
      required this.brojReda,
      required this.sjediste,
      this.terminId,
      required this.termin});

  factory Karta.fromJson(Map<String, dynamic> json) => _$KartaFromJson(json);
  Map<String, dynamic> toJson() => _$KartaToJson(this);
}
